src/Components/Header/Header.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import './Header.css'
+
+const Header = () => {
+  return (
+    <h1 className="header">Rowdy Taters!</h1>
+  )
+}
```

Comment:
We don't have anything else to add into our header at this time, but will keep in mind that this may grow and need some additional details added then